File: Accenture/15.py
n = int(input())
l = list(map(int , input().split()))

even = []
odd = []

for i in range(0 , n):
    if i%2 ==0:
        even.append(l[i])
    else:
        odd.append(l[i])

even.sort()
odd.sort()

for i in even:
    print(i , end=" ")
print()
for i in odd:
    print(i , end=" ")

File: Accenture/15.cpp
#include<iostream>
#include<vector>
#include<algorithm>
using namespace std;

int main(){
    int n;
    cout << "Enter the size of the array : ";
    cin >> n;
    vector<int>odd ;
    vector<int>even;


    for(int i = 0 ; i < n ; i++){
        int x;
        cin >> x;
        if(i%2==0){
            even.push_back(x);
        }else{
            odd.push_back(x);
        }
    }

    sort(even.begin() , even.end());
    sort(odd.begin()  , odd.end());

    for(auto x :even){
        cout << x << " ";
    }
    cout << endl;
    for(auto x : odd){
        cout << x << " ";
    }
}